### PR TITLE
Add Redshift SPLIT_PART function

### PR DESCRIPTION
**Pull request by OTA2000**
Implement Redshift [SPLIT_PART](https://docs.aws.amazon.com/redshift/latest/dg/SPLIT_PART.html) function

### CODE REVIEW

**Comment by OTA2000** [comment]
@googlebot I signed it!

**Comment by OTA2000** [comment]
@danieldeleo plz review this.

**Comment by OTA2000** [comment]
@danieldeleo I've pulled the latest master branch and added a few tests. Please review it.

File: udfs/migration/redshift/split_part.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+CREATE OR REPLACE FUNCTION rs.split_part(
+  string STRING,
+  delimiter STRING,
+  part INT64)
+AS (
+  CASE
+    WHEN part < 1 THEN ERROR('The part argument must be an integer greater than 1!')
+    WHEN ARRAY_LENGTH(SPLIT(string, delimiter)) < part THEN ""
+    ELSE SPLIT(string, delimiter)[OFFSET(part - 1)]
```
Comment (danieldeleo) [inline udfs/migration/redshift/split_part.sql:25]: Change "OFFSET(part - 1)" to "ORDINAL(part)"

File: udfs/migration/redshift/split_part.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ * limitations under the License.
+ */
+
+CREATE OR REPLACE FUNCTION rs.split_part(
+  string STRING,
+  delimiter STRING,
+  part INT64)
+AS (
+  CASE
+    WHEN part < 1 THEN ERROR('The part argument must be an integer greater than 1!')
```
Comment (danieldeleo) [inline udfs/migration/redshift/split_part.sql:23]: 'The part argument must be an integer greater than 0'

**Comment by danieldeleo** [comment]
Thank you for promptly making the changes!

**Pull request merged**